core/src/apps/cardano/__init__.py

```diff
@@ -6,6 +6,9 @@
 CURVE = "ed25519"
 SEED_NAMESPACE = [HARDENED | 44, HARDENED | 1815]
 
+MAINNET_PROTOCOL_MAGIC = 764824073
+TESTNET_PROTOCOL_MAGIC = 1097911063
+
```

Comment:
this imho deserves a `ProtocolMagic` enum: https://docs.python.org/3/library/enum.html - and you can save the use of "Mainnet", "testnet", ... strings and also helper dicts like `KNOWN_PROTOCOL_MAGICS` that way

Reply:
The strings are needed in the show_tx function. So the map is a helper for that. I don't even need to know if the id is known. I only need to know if the one I received is a testnet one or not. ... We'll discuss it offline.